configure.ac

```diff
@@ -799,6 +799,8 @@ PHP_ARG_ENABLE([debug],
 AS_VAR_IF([PHP_DEBUG], [yes], [
   PHP_DEBUG=1
   ZEND_DEBUG=yes
+  CFLAGS="$CFLAGS -U NDEBUG"
```

Comment:
Nit: The form `-UNDEBUG` / `-DNDEBUG` (without a space) is more common. Not sure this matters

Reply:
I don't think it matters, but I've changed it (no more space), except for Windows, where the existing defines already used a space (that is pretty inconsistent on Windows anyway).